.github/workflows/test_suite_mac.yml

```
@@ -68,15 +68,15 @@ jobs:
         path: artifact
     - name: Extract Webots
       run: |
-        hdiutil mount artifact/webots-R2020b-rev1.dmg
+        hdiutil mount artifact/webots-R2020b-rev2.dmg
```

```suggestion
        hdiutil mount artifact/webots*.dmg
```

FYI, I quickly checked on the mac and this should work :-)